client/state/data-layer/wpcom/sites/comments-tree/index.js

```diff
@@ -17,15 +17,15 @@ import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
 import { errorNotice } from 'state/notices/actions';
 import { getRawSite } from 'state/sites/selectors';
 
-const fetchCommentsTreeForSite = ( { dispatch }, action ) => {
+export const fetchCommentsTreeForSite = ( { dispatch }, action ) => {
 	const { siteId, status = 'unapproved' } = action.query;
 
 	dispatch(
 		http(
 			{
 				method: 'GET',
 				path: `/sites/${ siteId }/comments-tree`,
-				apiNamespace: 'wpcom/v2',
+				apiVersion: '1',
```

Comment:
I didn't find any code usages doing that. Note that the default version is also set without a leading `v` https://github.com/Automattic/wpcom.js/blob/master/index.js#L81

The request also looks correct.
<img width="737" alt="screen shot 2017-08-19 at 4 40 01 pm" src="https://user-images.githubusercontent.com/1270189/29490827-2e1066ae-84fd-11e7-8db6-e48ac216b974.png">